### DISCUSSION TITLE
No "Design Tab" anywhere

### DISCUSSION THREAD

**Discussion by yeoldegrove** [Budibase/budibase#2925] [Help]
I am a budibase newbie and watched this video here https://www.youtube.com/watch?v=hmmLdeY-5d4&t=101s.
It is talked about a "Design Tab" after selecting e.g. a table.
I do not see this "Design Tab" anywhere... What could be the issue?

I am running a docker-compose setup with this image from the official docs `budibase.docker.scarf.sh/budibase/apps`.
http://localhost:10000/builder/portal/settings/update shows me version 0.9.136

**Comment by mjashanks** [comment]
Hey @yeoldegrove thanks for pointing this out. The issue is that the video is out of date! We removed the Design tab some time ago, to simplify things.

We removed alot of the design properties, then moved the really important ones into `Settings`. Let me know if there is something that you need which is missing, or any other questions.
